assets/js/components/Sectors.vue

```diff
@@ -647,6 +647,17 @@ export default Vue.extend({
         .on('mouseover', this.tip.show)
         .on('mouseout', this.tip.hide);
 
+        aentered.append('use')
```

Comment:
how is `left: 50%; top: 50%; transform: translate(-50%,-50%); width: x%` a hack? no media queries needed. you just add to the `<chart-container>`, the svg is already absolutely positioned inside it.